README.md

```diff
@@ -1,7 +1,7 @@
 [![Build Status](https://github.com/brown-ccv/Chamber.jl/actions/workflows/test.yml/badge.svg)](https://github.com/brown-ccv/Chamber.jl/actions?query=workflows/test)
 [![][docs-stable-img]][docs-stable-url]
 [![][docs-dev-img]][docs-dev-url]
-
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/brown-ccv/Chamber.jl/blob/master/notebooks/notebook_for_chamber.ipynb)
```

Comment:
I think this badge should go in the notebook. In the readme you could add a section with a link to the notebook and perhaps also include the badge there.

Reply:
Like this one?
https://github.com/brown-ccv/Chamber.jl/blob/9315696429b7c6cda3f8e854341820f379d824e2/README.md?plain=1#LL120C1-L121C1

Reply:
I would list it as an item under the examples.